tests/benchmarks/run-benchmarks.sh

```diff
@@ -8,21 +7,47 @@ if ! which hyperfine > /dev/null 2>&1; then
     exit 1
 fi
 
+BAT=''
+for arg in "$@"; do
+	case "$arg" in
+		--system)  BAT="bat" ;;
+		--debug)   BAT="../../target/debug/bat" ;;
+		--release) BAT="../../target/release/bat" ;;
+		--bat=*)   BAT="${arg:6}" ;;
+	esac
+done
+
+if [[ -z "$BAT" ]]; then
+	echo "A build of 'bat' must be specified for benchmarking."
+	echo "You can use '--system', '--debug', or '--release'."
+	exit 1
+fi
+
+if ! command -v "$BAT" &>/dev/null; then
+	echo "Could not find the build of bat to benchmark."
+	case "$BAT" in
+		"bat")                        echo "Make you sure to symlink 'batcat' as 'bat'." ;;
+		"../../target/debug/debug")   echo "Make you sure to 'cargo build' first." ;;
+		"../../target/debug/release") echo "Make you sure to 'cargo build --release' first." ;;
```

Comment:
those paths seem wrong, right? There is one additional `/debug/`

Reply:
By the way, for me, those paths would not work. I have a custom
```toml
[build]
target-dir = "/home/shark/.cargo-target"
```
in `~/.cargo/config` (which I can only recommend).

Reply:
Fixed in 3ed0081f1fb4b707af026fcf1789909114487975.